### DISCUSSION TITLE
Using GPIO with Raspberry Pi?

### DISCUSSION THREAD

**Discussion by abishur** [frangoteam/FUXA#1233] [Q&A]
I just came across this project and it looks nice.  I was curious, if I install this on the Raspberry Pi (specifically thinking about the Pi Zero 2 W) is there any way to have it directly control the GPIO pins?

**Comment by unocelli** [comment]
Hi, I did some tests with the onoff library, and it worked well

**Reply by abishur** [reply]
With the onoff library, do I access the module through scripting or will it show as connection type option?  I've created a fork of the master branch of this repository, and in my fork in the server/package.json file added the "onoff" package as a dependency.  I changed the dockerfile to use the fork I made and built it, but I'm not sure how to access it at this point.

**Reply by unocelli** [reply]
I added as device, look in the 'raspy' branch
